### DISCUSSION TITLE
understanding viral-escape result data

### DISCUSSION THREAD

**Discussion by dddpower** [brianhie/viral-mutation#11] [Q&A] [answered]
Hi,
To make experiments I'm trying to run the codes, especially 'SARS-CoV-2 Spike Single-residue escape prediction'
(The required memory was so high to run the whole data), just to see the flow of execution and result data, I decreased the number of input data and batch size to make fast execution.
I finally got some result but since my majoring field is limited to computer science, I am having trouble to interpret the result data. Can I kindly ask you to how to interpret the result data? Here are my questions.

1. about the analyze_semantics_cov_bilstm_512.txt files
    I don't understand the meaning of 'pos', 'wt', 'mut', 'prob', 'change', 'is_viable', 'is_escape' fields. Can you please describe more of them?

2. how to generate pdb file. 
  analyze_semantics_cov_bilstm_512.txt  cov_mutations.txt were generated under target/cov/semantics/cache but I don't know how to generate pdb file. Is there any specific way to make it?

**Comment by brianhie** [comment, marked as answer]
Hi @dddpower,

1. The fields are, respectively, the index into the sequence, the wildtype residue, the mutant residue, the grammaticality score, the semantic change score, whether the mutant was observed in the DMS experiment, and whether the mutant is considered an escape mutant.
2. The PDB files are given; to color by escape, open the PDB files in PyMOL and run the corresponding PML script. E.g., all the files are here: https://github.com/brianhie/viral-mutation/tree/master/results/flu/structure

**Reply by dddpower** [reply]
Thank you!
